changelogs/fragments/2881-gitlab_project-fix_workspace_user.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - gitlab_project - user projects are created using namespace ID, instead of user ID (https://github.com/ansible-collections/community.general/pull/2881).
```

This is clearer in changelog IMO.
```suggestion
  - gitlab_project - user projects are created using namespace ID now, instead of user ID (https://github.com/ansible-collections/community.general/pull/2881).
```